frontend/src/App.js

```diff
@@ -14,28 +14,52 @@ import { CookiesProvider } from "react-cookie";
 import MakeList from './pages/MakeList';
 import MentorPage from './pages/MentorPage'
 
-function App() {
-  return (
-    <CookiesProvider>
-      <Router>
-        <Header />
-        <main className='py-3'>
-          <Container>
-            <Route path='/' component={LandingPage} exact />
-            <Route path='/curated' component={HomePage} exact />
-            <Route path='/curated/:id' component={ListPage} exact />
-            <Route path='/mentors' component={MentorsPage} exact />
-            <Route path='/login' component={LoginPage} exact />
-            <Route path='/signup' component={SignupPage} exact />
-            <Route path='/profile' component={ProfilePage} exact />
-            <Route path='/mentor/:id' component={MentorPage} exact />
-            <Route path='/profile/makelist' component={MakeList} exact />
-          </Container>
-        </main>
-        <Footer />
-      </Router>
-    </CookiesProvider>
-  );
+import { instanceOf } from 'prop-types';
+import { withCookies, Cookies } from 'react-cookie';
+
+class App extends Component {
+  static propTypes = {
+    cookies: instanceOf(Cookies).isRequired
+  };
+  constructor(props) {
+    super(props);
+    const { cookies } = props;
+    this.state = {
+        loggedin: cookies.get('loggedin') || cookies.set('loggedin', "abcd", '/'),
+        uid: cookies.get('uid') || cookies.set('uid', "1234", '/')
+    };
+}
+  // componentDidMount(loggedin,uid, cookies){
```

Comment:
Should we remove this commented code entirely if we're not using it?

Reply:
oops forgot about that
I'll do it